staff/staff_views_helper.py

```diff
@@ -159,19 +178,35 @@ def get_emails_from_form(form):
         )
         for student in students:
             email_list.append(student.email)
+            user_list.append(student)
         for volunteer in volunteers:
             email_list.append(volunteer.email)
+            user_list.append(volunteer)
+    return {"email_list": email_list,
+            "user_list": user_list}
+
+
+def get_emails_from_form_distributions(form_distributions):
+    email_list = []
+    for form_dist in form_distributions:
+        user_email = DjangoUser.objects.get(id=form_dist.user_id).email
+        email_list.append(user_email)
     return email_list
 
 
 def email_announcement(request, form, email_list):
-    subject = form.instance.title
+    try:
```

Comment:
This is a bit unconventional, maybe instead of taking a form it should take the subject and message directly? The implementation doesn't use the form for anything else. There's only two call sites so it wouldn't really add anything redundant.

```python
def email_announcement(request, subject, message, email_list):
```

Reply:
Yes, good catch, fixed in 0e9f3166a818d89d42c098c9bb90f076087cdd93